.travis.yml

```diff
@@ -25,12 +28,24 @@ before_install:
   - ./runflake8
   - ./runisort
 
+  # Install NVM:
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
+
+  # Install nodeJS:
+  - nvm install v8
+
 install:
   - pip install $DJANGO
   - python setup.py -q develop
 
+  # Install JSlint:
+  - npm install --save jslint -g
+
 script:
   - coverage run --source=django_netjsongraph runtests.py
 
+  # JSlint check:
+  - jslint --config ./.jslintrc ./django_netjsongraph/static/netjsongraph/js/*.js
```

Comment:
move this in before_install section, after installing jslint
we do this to stop the build if any style check fails before installing the other python dependencies, which speeds up a lot the style checks and failures are reported very early if they happen